exercises/pig-latin/src/example.c

```diff
@@ -0,0 +1,123 @@
+#include "pig_latin.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+static bool is_vowel(const char letter)
+{
+   return letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o'
+       || letter == 'u';
+}
+
+static bool starts_with_vowel(const char letter_a, const char letter_b)
+{
+   return is_vowel(letter_a) || (letter_a == 'x' && letter_b == 'r')
+       || (letter_a == 'y' && letter_b == 't');
+}
+
+static void count_words_and_chars(const char *input, size_t * words,
+                                  size_t * chars)
+{
+   bool in_word = false;
+   while (*input) {
+      (*chars)++;
+      if (isalpha(*input++)) {
+         *words += in_word ? 0 : 1;
+         in_word = true;
+      } else {
+         in_word = false;
+      }
+   }
+}
+
+static void mstrncpy(char *dst, const char *src, size_t n)
```

Comment:
What does the `m` mean?

Reply:
Named after strncpy because it does the same up to the 'tolower' part. The ```m``` (my) was to avoid confusion  with the "real" strncpy. I suppose now it is still a confusing name ... :/